polkadot/primitives/src/vstaging/mod.rs

```diff
@@ -430,49 +434,45 @@ pub enum UMPSignal {
 /// Separator between `XCM` and `UMPSignal`.
 pub const UMP_SEPARATOR: Vec<u8> = vec![];
 
-impl CandidateCommitments {
-	/// Returns the core selector and claim queue offset the candidate has committed to, if any.
-	pub fn selected_core(&self) -> Option<(CoreSelector, ClaimQueueOffset)> {
-		// We need at least 2 messages for the separator and core selector
-		if self.upward_messages.len() < 2 {
-			return None
-		}
-
-		let separator_pos =
-			self.upward_messages.iter().rposition(|message| message == &UMP_SEPARATOR)?;
-
-		// Use first commitment
-		let message = self.upward_messages.get(separator_pos + 1)?;
+/// Utility function for skipping the ump signals.
```

Comment:
Any good reason not to put it inside `CandidateCommitments` ?

Reply:
yes, in the runtime we use this function after taking the upward messages from the commitments struct